app/controllers/observations_controller.rb

```diff
@@ -2113,8 +2114,10 @@ def retrieve_photos(photo_list = nil, options = {})
         photo = if photo_class == LocalPhoto
           if photo_id.is_a?(Integer) || photo_id.is_a?(String)
             LocalPhoto.find_by_id(photo_id)
-          else
-            LocalPhoto.new(:file => photo_id, :user => current_user) unless photo_id.blank?
+          elsif !photo_id.blank?
+            lp = LocalPhoto.new( user: current_user )
+            lp.file = photo_id
+            lp
```

Comment:
This stuff isn't related to the bucket changing, right?

Reply:
I think this was related to a previous approach I tried. I've reverted this